docs_outdoornav_user_manual/api/api_examples/mission_with_custom_tasks.mdx

```diff
@@ -0,0 +1,235 @@
+---
+title: Mission with Custom Tasks
+sidebar_label: Mission with Custom Tasks
+sidebar_position: 3
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+Before being able to run the examples similar to the one explained below, it is reuired that you have written your own custom tasks. See the
+[Custom Tasks](/docs_outdoornav_user_manual/features/custom_tasks) section for information on how to develop tasks for your application.

```

Comment:
I think we need to do links differently now to ensure that you get the link for the matching release version. Probably ../../features/custom_tasks

Reply:
Agreed. I think it probably worth changing all the links in such a way like how you mentioned. I will change all of the links to match this convention then.

Reply:
making a seperate ticket to confirm the links are correct